src/control/server/README.md

```diff
@@ -1,431 +1,204 @@
-wolf-72.wolf.hpdd.intel.com 2019/04/10 04:00:22 external.go:98: debug: calling mount with /dev/pmem4, /mnt/daos, ext4, 33806, dax
-wolf-72.wolf.hpdd.intel.com 2019/04/10 04:00:22 external.go:54: debug: exec 'mount | grep ' /mnt/daos ''
-wolf-72.wolf.hpdd.intel.com 2019/04/10 04:00:22 mgmt.go:153: debug: StorageFormat: storage format successful on server 0
-wolf-72.wolf.hpdd.intel.com 2019/04/10 04:00:22 iosrv.go:128: debug: format server /mnt/daos (createMS=false bootstrapMS=false)
-wolf-72.wolf.hpdd.intel.com 2019/04/10 04:00:23 main.go:156: debug: DAOS server listening on 0.0.0.0:10001
-DAOS I/O server (v0.4.0) process 135863 started on rank 0 (out of 1) with 1 target xstream set(s), 2 helper XS, firstcore 0.
-```
-
-</p>
-</details>
-
-#### SCM Provisioning
-
-Provisioning SCM occurs by configuring DCPM modules in AppDirect memory regions (interleaved mode) in groups of modules local to a specific socket (NUMA) and resultant nvdimm namespaces are defined a device identifier (e.g. /dev/pmem0). This can be performed through the [ipmctl](https://github.com/intel/ipmctl) tool.
-
-TODO: implement set-interleaved command
-
-### NVMe management capabilities
-
-Operations on NVMe SSD devices are performed using [go-spdk bindings](https://github.com/daos-stack/go-spdk) to issue commands over the SPDK framework.
-
-The DAOS data plane utilises NVMe devices as block storage using the SPDK "[blobstore](https://spdk.io/doc/blob.html)" abstraction, creation and management of the blobstore and blobs is performed by the data plane.
-
-The DAOS control plane will provide NVMe storage management capabilities enabling the discovery, format, initial burn-in testing, firmware update and allocation of devices to data plane instances.
-
-#### NVMe Controller and Namespace Discovery
-
-Device details for any discovered NVMe SSDs accessible through SPDK on the storage server will be returned when running `storage scan` subcommand on [`dmg`](../cmd/dmg/README.md#subcommands) or [`daos_server`](#storage-list) executables.
-
-The following animation illustrates starting the control server and using the management shell to view the NVMe Namespaces discovered on a locally available NVMe Controller (assuming the quickstart guide instructions have already been performed):
-
-![Demo: List NVMe Controllers and Namespaces](/doc/graph/daosshellnamespaces.svg)
+If storage has not been previously formatted, `daos_server` will
+halt on start-up waiting for storage format to be
+[triggered](/src/control/cmd/dmg/README.md) through the management
+tool.
+
+Storage format is expected only to be performed when installing DAOS
+system for the first time.
+
+#### SCM Format
+
+Formatting SCM involves creating an ext4 filesystem on the nvdimm
+device.  Mounting SCM results in an active mount using the DAX
+extension enabling direct access without restrictions imposed by
+legacy HDD hardware.
```

Comment:
Maybe instead of "legacy HDD hardware," something like "traditional block storage"